packages/common/src/api/search.ts

```diff
@@ -1,6 +1,22 @@
 import { createApi } from '~/audius-query'
+import { ModalSource } from '~/models'
 import { ID } from '~/models/Identifiers'
 import { SearchKind } from '~/store/pages/search-results/types'
+import { Genre } from '~/utils'
+
+export type SearchCategory = 'all' | 'tracks' | 'albums' | 'playlists' | 'users'
+
+export type SearchFilters = {
+  genre?: Genre
+  mood?: ModalSource
+  bpm?: string
+  key?: string
+  isVerified?: boolean
+  hasDownloads?: boolean
+  isPremium?: boolean
```

Comment:
do we want to include the sort options here or separately?

Reply:
i left them out since it seems we dont have them for mobile? when we find out can add them!